packages/wonder-blocks-announcer/src/__tests__/util/util.test.ts

```diff
@@ -0,0 +1,39 @@
+import {debounce} from "../../util/util";
+
+describe("Debouncing messages", () => {
+    jest.useFakeTimers();
+
+    test("a single message", async () => {
+        // ARRANGE
+        const callback = jest.fn((message: string) => message);
+        const debounced = debounce(callback, 300);
+
+        // ACT
+        const resultPromise = debounced("Hello, World!");
+
+        // ASSERT
+        expect(resultPromise).toBeInstanceOf(Promise);
```

Comment:
**question:** Is this assertion required for this test? I wonder if just using the expectation below is a good signal to see if the test works as expected?

Reply:
Makes sense to me!